planetaryimage/pds3image.py

```diff
@@ -123,6 +136,15 @@ def _save(self, file_to_write, overwrite):
         image_pointer = int(label_sz / self.label['RECORD_BYTES']) + 1
         self.label['^IMAGE'] = image_pointer + 1
 
+        if self._sample_bytes != self.data.itemsize:
+            self.label['IMAGE']['SAMPLE_BITS'] = self.data.itemsize * 8
+            sample_type_to_save = self.DTYPES[self.dtype.byteorder + self.dtype.char]
+            self.label['IMAGE']['SAMPLE_TYPE'] = sample_type_to_save
+
+        self.label['IMAGE']['BANDS'] = self.shape[0]
```

Comment:
If Image Shape is changed, these 3 lines take care of changing the label information.